osu.Game/Overlays/SkinEditor/SkinEditor.cs

```diff
@@ -263,19 +266,67 @@ public void UpdateTargetScreen(Drawable targetScreen)
 
             void loadBlueprintContainer()
             {
-                Debug.Assert(content != null);
+                selectedTarget.Default = getFirstTarget()?.Lookup;
+
+                if (!availableTargets.Any(t => t.Lookup.Equals(selectedTarget.Value)))
+                    selectedTarget.Value = getFirstTarget()?.Lookup;
+                else
+                    selectedTarget.TriggerChange();
```

Comment:
Not super sure what this `TriggerChange()` is supposed to be doing - mind explaining?

Reply:
I'm 99% sure this was only relevant in a more isolated change set, and is no longer required. Will remove.